src/Tickets/Seating/Admin/Ajax.php

```diff
@@ -1033,4 +1039,87 @@ public function handle_seat_type_deleted() {
 			]
 		);
 	}
+
+	/**
+	 * Updates the layout of an event.
+	 *
+	 * @since TBD
+	 *
+	 * @return void The function does not return a value but will echo the JSON response.
+	 */
+	public function update_event_layout() {
+		if ( ! $this->check_current_ajax_user_can( 'edit_posts' ) ) {
+			return;
+		}
+		
+		$post_id   = tribe_get_request_var( 'postId' );
+		$layout_id = tribe_get_request_var( 'newLayout' );
+		
+		if ( empty( $layout_id ) || empty( $post_id ) ) {
+			wp_send_json_error(
+				[
+					'error' => 'No layout ID or post ID provided',
+				],
+				400
+			);
+			
+			return;
+		}
+		
+		$layout = DB::table( \TEC\Tickets\Seating\Tables\Layouts::table_name( false ) )->where( 'id', $layout_id )->get();
+		
+		/** @var \Tribe__Tickets__Tickets_Handler $tickets_handler */
+		$tickets_handler   = tribe( 'tickets.handler' );
+		$capacity_meta_key = $tickets_handler->key_capacity;
+		
+		$seat_types        = $this->seat_types->get_in_option_format( [ $layout_id ] );
+		$new_seat_type_id  = $seat_types[0]['id'];
+		$new_seat_capacity = $seat_types[0]['seats'];
+		
+		$updated_tickets   = 0;
+		$updated_attendees = 0;
+		
+		// Get tickets by post id.
+		$tickets = tribe_tickets()->where( 'event', $post_id )->get_ids( true );
+		
+		foreach ( $tickets as $ticket_id ) {
+			clean_post_cache( $ticket_id );
```

Comment:
Do not do this here, do this in the tests. It's fine if the meta value is cached.